packages/coin-support-utils/tests/02.createAccount/01.createDerivationPathGenerator/index.ts

```diff
@@ -0,0 +1,14 @@
+import { describe, test } from '@jest/globals';
+import { createDerivationPathGenerator } from '../../../src/createAccount';
+import { fixtures } from './__fixtures__';
+
+describe('01. Create Derivation Path Generator', () => {
```

Comment:
If the test file is for testing a particular function, try to keep the description as the function name itself.  
```suggestion
describe('createDerivationPathGenerator', () => {
```